spec/api-app-spec.js

```diff
@@ -289,4 +289,37 @@ describe('app module', function () {
       })
     })
   })
+
+  describe('app.launcher API', function() {
+    it('should be available on linux', function() {
+      if (process.platform !== 'linux') {
+        assert.equal(app.launcher, undefined);
```

Comment:
Actually I didn't remember to run the linter...
Rebased, Fixed the errors, squashed it into the specs commit :smile: